runtime/resolvers/clickhouse_resolvers_test.yaml

```diff
@@ -0,0 +1,91 @@
+project:
```

Comment:
Please move the `.yaml` test fixtures into a separate directory at `runtime/resolvers/testdata` since the Go docs recommend putting non-Go test files in a separate `testdata` directory

Reply:
Done.